EIPS/eip-5089.md

```diff
@@ -0,0 +1,391 @@
+---
+eip: 5089
+title: Principal Token
+description: Principal tokens (zero-coupon tokens) are redeemable for a single underlying ERC-20 token at a future timestamp.
+author: Julian Traversa (@JTraversa), Robert Robbins (@robrobbins), Alberto Cuesta Cañada (@alcueca)
+discussions-to: https://ethereum-magicians.org/t/eip-5089-principal-token-standard/9259
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-05-01
+requires: 20, 2612
+---
+
+## Abstract
+
+The following specification is an API for principal tokens --
+representing ownership of an underlying [ERC-20](./eip-20.md) token at a future timestamp.
```

Comment:
```suggestion
representing ownership of an underlying [EIP-20](./eip-20.md) token at a future timestamp.
```
All EIPs should be referenced and linked as `EIP-XXX`.  ERC is a subtype, and in the context of this repository it is not a separate prefix.  Same feedback throughout the document.